README.md

```diff
@@ -57,7 +57,7 @@ Inspired by the [awesome](#more-awesome) list thing. Feel free to <a href="https
 * [Pokelmon](https://github.com/brenopanzolini/pokelmon) - Elm project consuming PokéAPI.
 * [JWT auth with Django + Elm](https://github.com/apirobot/django-elm-auth-with-jwt) - JSON Web Token (JWT) authentication using Django (backend) and Elm (frontend).
 * [Bitcoin BR Chrome Extension](https://github.com/jouderianjr/bitcoin-br-chrome-extension) - Chrome extension built in Elm that shows the bitcoin value in all Brazilian exchanges.
-* [Elmstagram](https://github.com/bkbooth/Elmstagram) - Basic UI Clone of Instagram / Articles - [Part 1](https://benkbooth.com/building-a-basic-ui-clone-of-instagram-using-elm-part-1/) [Part 2](https://benkbooth.com/post/building-a-basic-ui-clone-of-instagram-using-elm-part-2/) [Part 3](https://benkbooth.com/post/building-a-basic-ui-clone-of-instagram-using-elm-part-3/)
+* [Elmstagram](https://github.com/bkbooth/Elmstagram) - Basic UI Clone of Instagram / Articles - [Part 1](https://benbooth.dev/building-a-basic-ui-clone-of-instagram-using-elm-part-1/) [Part 2](https://benbooth.dev/building-a-basic-ui-clone-of-instagram-using-elm-part-2/) [Part 3](https://benbooth.dev/building-a-basic-ui-clone-of-instagram-using-elm-part-3/)
```

Comment:
The current link https://benkbooth.com/building-a-basic-ui-clone-of-instagram-using-elm-part-1 appears to re-direct to https://benbooth.dev/building-a-basic-ui-clone-of-instagram-using-elm-part-1
Might as well update the link. 
![image](https://user-images.githubusercontent.com/194400/67315271-1e602a00-f4fe-11e9-97c6-f471a6f63b4f.png)

👍

Reply:
The part 1 link worked but parts 2 and 3 were broken.
https://benkbooth.com/post/building-a-basic-ui-clone-of-instagram-using-elm-part-2/

Reply:
Confirmed. ✅ 
https://benbooth.dev/post/building-a-basic-ui-clone-of-instagram-using-elm-part-2
![image](https://user-images.githubusercontent.com/194400/67321791-ea8a0200-f507-11e9-81d5-530b62707776.png)

New link: https://benbooth.dev/building-a-basic-ui-clone-of-instagram-using-elm-part-2
![image](https://user-images.githubusercontent.com/194400/67321914-1f965480-f508-11e9-9fd8-28c59008d5db.png)


https://benbooth.dev/post/building-a-basic-ui-clone-of-instagram-using-elm-part-3
![image](https://user-images.githubusercontent.com/194400/67321826-fa094b00-f507-11e9-879c-259e6cd20f3d.png)

New link: https://benbooth.dev/building-a-basic-ui-clone-of-instagram-using-elm-part-3
![image](https://user-images.githubusercontent.com/194400/67321946-2d4bda00-f508-11e9-8138-d9cf27d52c2a.png)

Good PR. 👍